eventq_rabbitmq/lib/eventq_rabbitmq/rabbitmq_queue_worker.rb

```diff
@@ -28,7 +28,7 @@ def start(queue, options = {}, &block)
         raise "[#{self.class}] - Worker is already running." if running?
 
         if options[:client] == nil
```

Comment:
on reflection I think we should allow both singleton and instance per thread approaches and not deprecate the singleton client passed via options. As to my knowledge this has not caused us any issues? the issue that is driving this change is due to the usage of curb in an azure queue client implementation as curb is not thread safe.